compose-files/docker-compose.yml

```diff
@@ -89,8 +87,8 @@ services:
     ports:
       - "5432:5432"
     volumes:
-      - 'data-calendar-postgres:/var/lib/postgresql/data'
-      - 'postgres-init:/docker-entrypoint-initdb.d'
+      - "data-calendar-postgres:/var/lib/postgresql/data"
+      - "postgres-init:/docker-entrypoint-initdb.d"
```

Comment:
Why?

Reply:
I understood your last comment as `we prefer " over '`, but I can change hose back who were single quotes before this PR

Reply:
It's nice that you prefer it but no Scout rule.
If it doesn't need to touch don't touch it.
It mess up my ability to track changes and blame Teams for it

Reply:
Of course, will change it back :)